src/main/java/com/puppycrawl/tools/checkstyle/checks/javadoc/JavadocTagContinuationIndentationCheck.java

```diff
@@ -50,6 +50,16 @@
  * <pre>
  * &lt;module name="JavadocTagContinuationIndentation"/&gt;
  * </pre>
+ * <pre>
+ * &#47;**
+ *  * @tag comment..
```

Comment:
`comment..` no need for 2 dot at the end

Reply:
done